src/__tests__/standalone.test.js

```diff
@@ -341,12 +341,27 @@ describe("standalone", () => {
   });
 
   it("should change unicode symbols in the result", async () => {
+    const result = await standalone({
+      files: `${fixturesGlob}/svg-icons/**/*`,
+      formats: ["eot"],
+      glyphTransformFn: obj => {
+        obj.unicode = ["\u0001"];
+
+        return Promise.resolve(obj);
```

Comment:
We can avoid this here, because return in promise works like Promise.resolve

Reply:
The problem here is eslint, I didn't want to change the rules.

```js
glyphTransformFn:  async obj => { // eslint complains that no await is used
    obj.unicode = ["\u0001"];

    return obj;
}
```

Reply:
remove this from test too